src/styles/blocks/header.scss

```diff
@@ -33,10 +33,16 @@
   }
 
   &__link {
+    @extend %smooth-transition;
+
     color: inherit;
     text-decoration: none;
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    &:hover {
+      color: #00acdc;
+    }
```

Comment:
you need to apply this to the item, not to the link, because the hover should not only work when you point at the text itself, but also below and above it

Reply:
Its logically wrong to change link style when the user hover on the item, but i get idea about that hover must work on all height of panel, so thanks for suggestion.